ui/components/component-library/form-text-field/form-text-field.js

```diff
@@ -10,7 +10,7 @@ import {
 
 import Box from '../../ui/box/box';
 
-import { TextField } from '../text-field';
+import { TextField, TextFieldSize, TextFieldType } from '../text-field';
```

Comment:
Can we use the deprecated version here as well and revert all the propType changes? So the only changes to this file will be the import path of the TextField e.g.
```
import { TextField } from '../text-field/deprecated';
```